src/components/header/Nav/Nav.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from 'react';
+import { getUser, getUserById } from '@/pages/api/api';

```

Comment:
경로 문제로 인한 타입 해결 필요합니다~!
```suggestion
import { getUser, getUserById } from '@/pages/api/hello';
```